basics-assignments-4-start/src/app/my-inner/my-inner.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Output, EventEmitter }from '@angular/core';
+
+@Component({
+  selector: 'app-my-inner',
+  templateUrl: './my-inner.component.html',
+  styleUrls: ['./my-inner.component.css']
+})
+export class MyInnerComponent {
+  @Output() decreasedInput: EventEmitter<number> = new EventEmitter<number>();
+  @Output() increasedInput: EventEmitter<number> = new EventEmitter<number>();
+  innerTotal: number = 5;
+  addPoint(){
```

Comment:
No return type defined.
Between class variables and functions make sure to keep an empty line